index.html

```diff
@@ -136,75 +144,43 @@ <h2>Multi-Post <br> Stories</h2>
                 <li>2015</li>
             </ul>
             <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <div class="box">
-                <div><a href="">html</a></div>
-                <div><a href="">css</a></div>
-                <div><a href="">javaScript</a></div>
-            </div>
-            <button><a href="">See Project</a></button>
+            <ul>
+                <li><a href="">html</a></li>
+                <li><a href="">css</a></li>
+                <li><a href="">javaScript</a></li>
+            </ul>
+            <button type="button"><a href="">See Project</a></button>
         </div>
-    </section>
+    </div>
 
 
-    <div class="about-myself-container">
+    <div class="about-section">
```

Comment:
- [ ] Kindly make sure to wrap `About myself` in the `section` tag and make sure to use CSS `flex-box` to layout it's contents on the page in order to meet the project requirements. :wink: